content/12.privacy.md

```diff
@@ -1 +1,8 @@
 ## Rule 10: Don't share models trained on sensitive data
+
+One of the greatest opportunities for deep learning in biology is the high dimensional nature of much of the biological data collected today. This abundance of features for each sample means that the models can learn an enormous amount about the data during the training process, and therefore it is important not to share models trained on sensitive data. Applying deep learning to images of cats from the internet does not pose a significant privacy problems, but in the field of human health, this starts to approach privacy concerns. 
```

Comment:
Thanks for the contribution.  Can you please switch to one sentence per line?  That makes it easier to give specific feedback in the review.

Reply:
Its been changed